pkg/hash/sha-2.go

```diff
@@ -0,0 +1,11 @@
+package hash
+
+import (
+	"crypto/sha256"
+	"fmt"
+)
+
+func Hash(someString string) string {
```

Comment:
It would be nice to add a comment with a short description.

Reply:
Comment with a short description added